packages/skeleton-common/src/classes/menu.ts

```diff
@@ -0,0 +1,17 @@
+import { defineSkeletonClasses } from '../internal/define-skeleton-classes.js' with { type: 'macro' };
+
+export const classesMenu = defineSkeletonClasses({
+	trigger: '',
+	contextTrigger: '',
+	positioner: '',
+	indicator: '',
+	content: 'card bg-surface-50-950 border border-surface-200-800 p-2 flex flex-col gap-2',
+	arrow: '',
+	arrowTip: '',
+	itemGroup: '',
+	itemGroupLabel: 'text-surface-600-400 text-xs px-2 py-1',
+	item: ['flex justify-between items-center px-2 py-1 rounded cursor-pointer', 'data-highlighted:preset-tonal'],
```

Comment:
If formatting is going to break wrapping, just combine these to a single string line.